tracing-opentelemetry/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.12.0 (March 31, 2021)
+
+### Breaking Changes
```

Comment:
there's also a performance improvement in this release, right (#1315)? can we add that to the changelog?

Reply:
can do!